src/Controller/UserController.php

```diff
@@ -125,16 +153,28 @@ public function newUser(User $user, EntityManagerInterface $manager, ValidatorIn
         $manager->persist($user);
         $manager->flush();
 
-        return $user;
+        $keyAll = 'user.all?page=' . $request->query->getInt('page', 1);
+        $keyOnce = 'user.once';
+
+        $cacheAll = $this->adapter->getItem($keyAll);
+        $cacheOnce = $this->adapter->getItem($keyOnce);
+
+        if (true === $cacheAll->isHit()){
```

Comment:
tu fais deux fois la même condition

Reply:
No, this action is applicated to 2 different keys

Reply:
Ah oui, désolé ^^